snmp/datadog_checks/snmp/data/default_profiles/dell-poweredge.yaml

```diff
@@ -76,6 +76,7 @@ metrics:
           4: non_critical
           5: critical
           6: non_recoverable
+          7: absent
```

Comment:
New metric

Reply:
It's probably fine to not have a bug fix since it won't be a breaking change to add the new mapping later.

Will be fixed for 7.49.